content/applications/sales/sales/send_quotations/pdf_quote_builder.rst

```diff
@@ -0,0 +1,216 @@
+=================
+PDF quote builder
+=================
+
+The *PDF Quote Builder* in Odoo *Sales* provides the opportunity to send customers a fully
+customized PDF file for quotes, showcasing the company and products, with various information and
+design elements, instead of just showing the price and total.
+
+The PDF Quote Builder groups header pages, product descriptions, the price(s), and footer pages
+to create a detailed quote. It can also inject dynamic texts in the PDF to personalize the offer
+for the customer.
+
+Having a customized PDF in quotes provides a heightened conclusion to the shopping experience for
+customers, and adds an elegant level of professionalism to a company.
+
+Configuration
+=============
+
+In order to add custom PDF files for quotes, the :guilabel:`PDF Quote builder` feature *must* be
+configured.
+
+To do that, navigate to :menuselection:`Sales app --> Configuration --> Settings`. Then, on the
+:guilabel:`Settings` page, scroll to the :guilabel:`Quotations & Orders` section, and locate the
+:guilabel:`PDF Quote builder` feature.
+
+.. image:: pdf_quote_builder/pdf-quote-builder-feature.png
+   :align: center
+   :alt: The PDF Quote Builder feature located on the Settings page of the Sales application.
+
+Here, custom :guilabel:`Header pages` and :guilabel:`Footer pages` can be uploaded. To upload
+either, click the :guilabel:`Upload your file` button, or the :guilabel:`✏️ (pencil)` icon to the
+right of the desired field, and proceed to locate, select, and upload the desired PDF file.
+
+.. note::
+   Headers and footers can also be added directly in a quotation template, so it's possible to have
+   different headers/footers on a quotation template(s).
+
+Clicking the :guilabel:`🗑️ (trash)` icon deletes the current PDF file and replaces the blank field
+with an :guilabel:`Upload your file` button.
+
+Once the desired PDF file(s) are uploaded in the appropriate fields in the :guilabel:`PDF Quote
+builder` section of the *Sales* :guilabel:`Settings` page, be sure to click :guilabel:`Save`.
+
+The files uploaded here will be the default PDF used for all quotes.
+
+.. note::
+   Values set in the PDF Quote Builder settings are company-specific.
+
+Dynamic text in PDFs
+====================
+
+While creating custom PDFs for quotes, use *dynamic text* for Odoo to auto-fill the PDF content with
+information related to the quote from the Odoo database, like names, prices, etc.
+
+Dynamic text values are form components (text inputs) that can be added in a PDF file, and Odoo
+automatically fills it in with information related to the quote. Odoo fills these components
+according to their name. Doing so streamlines the writing of personalized quotations.
```

Comment:
- Mixed subject in the first sentence
- don't need 2nd or 3rd sentence after that

```suggestion
Dynamic text values are form components (text inputs) that can be added in a PDF file, and Odoo
automatically fills those values in with information related to the quote.
```